Add splashboard (#319)
ratatui/awesome-ratatui: README.md

- [scriptor](https://github.com/giacomopiccinini/scriptor) - A (medieval) local speech-to-text TUI & CLI.
- [snipt](https://github.com/snipt/snipt) - A text snippet expansion tool with a TUI for managing snippets.
- [solverforge-calendar](https://github.com/blackopsrepl/solverforge-calendar) - A TUI calendar with Google Calendar sync and DAG-linked events.

> - [splashboard](https://github.com/unhappychoice/splashboard) - A customizable terminal splash rendered on shell startup and on directory change, with per-directory dashboards.

Reviewer: Let's shorten the description a bit:

```suggestion
- [splashboard](https://github.com/unhappychoice/splashboard) - A customizable terminal splash rendered on shell startup or directory updates.
```

Just a suggestion.
Feel free to shorten it in the way you want as well :)
